spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/MockMvcMultipartGiven.java

```diff
@@ -68,9 +70,14 @@ private String getMultipartParameterLine(SingleContractMetadata metadata,
 
 	private String getMultipartFileParameterContent(SingleContractMetadata metadata,
 			String propertyName, NamedProperty propertyValue) {
-		return getJavaMultipartFileParameterContent(propertyName, propertyValue,
-				fileProp -> this.bodyReader.readBytesFromFileString(metadata, fileProp,
-						CommunicationType.REQUEST));
+		// For now, I don't like this :(
+		return KotlinClassMetaData.hasKotlinSupport()
```

Comment:
Again, I would opt for `KotlinMockMvcMultipartGiven`, etc. @marcingrzejszczak ?

Reply:
Should have been removed